WooCommerce/Classes/POS/Presentation/Barcode Scanner Setup/PointOfSaleBarcodeScannerSetupFlow.swift

```diff
@@ -347,7 +398,8 @@ private extension PointOfSaleBarcodeScannerSetupFlow {
             comment: "Title for the back button in barcode scanner setup navigation"
         )
         //TODO: WOOMOB-792
-        static let starSetUpBarcodeStepTitleFormat = "%1$@ Setup"
-        static let setUpBarcodeStepInstruction = "Scan the barcode to set up your scanner."
+        static let starSetUpBarcodeStepTitleFormat = "Scanner Setup"
```

Comment:
nit: it's now just a string literal, maybe the name & use cases can be updated?

Reply:
Also, this name suggests it's for the "Star scanner" (unless "star" means something else), while it's used in the Tera setup. Just another consideration for renaming this constant.